system/core/dynamic/BeanPopulator.cfc

```diff
@@ -442,4 +469,20 @@ Description :
 		<cfreturn createObject("component","coldbox.system.core.util.Util")/>
 	</cffunction>
 
+	<!--- Get the Target's properties as a struct with a key and value that are the same for easy access and setting of the key's casing --->
+	<cffunction name="getTargetPropertiesAsStruct" access="private" output="false" returntype="struct" hint="Return the property names for the target object as a struct with the key the same as the value">
```

Comment:
What about inheritance chains? Can you update this to search up the `extends` tree?